### PR TITLE
Serializer groups support

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

Enable serializer groups (related to symfony/symfony#13107).

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ framework:
     form:            ~
     csrf_protection: ~
     validation:      { enable_annotations: true }
+    serializer:      { enable_annotations: true }
```
Comment (fabpot) [inline app/config/config.yml]: I think it should be commented by default.

Comment (dunglas) [inline app/config/config.yml]: done

File: app/config/config_prod.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ imports:
 
 #framework:
 #    validation:
-#        cache: apc
+#        cache: validator.mapping.cache.apc
+#    serializer:
+#        cache: serializer.mapping.cache.apc
```
Comment (xabbuh) [inline app/config/config_prod.yml]: Will this service be available by default?

Comment (dunglas) [inline app/config/config_prod.yml]: Yes: https://github.com/symfony/symfony/pull/13107/files#diff-6cd5bb767c43a9c44442521122006d29R39

Comment (xabbuh) [inline app/config/config_prod.yml]: Thanks for pointing me in the right direction.

**Comment by dunglas** [comment]
ping @symfony/deciders

**Comment by aitboudad** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request closed without merging**